bindep.txt

```diff
@@ -2,8 +2,3 @@
 # see https://docs.openstack.org/infra/bindep/ for additional information.
 
 gcc-c++ [doc test platform:rpm]
-python3-devel [test platform:rpm]
-python3 [test platform:rpm]
-libssh-devel [test platform:rpm]
-libffi-devel [test platform:rpm]
-openssl-devel [test platform:rpm]
```

Comment:
Shouldn't these changes be mentioned in the changelog as well?

Reply:
I believe it is expected to at least have python3 available on a container image that is to be used as an EE (built via builder). So, I'm not sure if this really warrants a changelog. The other ones are _only_ required for using pylibssh transport so I've added them back.

Reply:
I agree that `python3` shouldn't be in this list :)